config/initializers/content_security_policy.rb

```diff
@@ -23,3 +23,7 @@
 # For further information see the following documentation:
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only
 # Rails.application.config.content_security_policy_report_only = true
+
+Rails.application.config.content_security_policy do |policy|
+  policy.connect_src :self, :https, "http://localhost:3035", "ws://localhost:3035" if Rails.env.development?
+end
```

Comment:
What's this for?

Reply:
Probably this is for `webpack-dev-server`. But please make sure app works well also without it as right now.

Reply:
it's not needed 👍🏻